src/clusterfuzz/_internal/bot/tasks/utasks/uworker_io.py

```diff
@@ -279,31 +265,42 @@ class UworkerOutput:
   """Convenience class for results from uworker_main. This is useful for
```

Comment:
Question for the purpose of this class. We're wrapping a proto, which already is a Python object. Could we instead just use that proto object directly and avoid this setattr hackery?

Reply:
We can't. There are fields in the proto that are serialized db entities but are not db entities. To use those entities we need to deserialize them from the proto format into a real python object. But we can't simply reassign the field to the deserialized version of the field because proto enforces types.
Did I explain this well? Should I leave this in a comment.

Reply:
Maybe another ignorant question but can we make the proto type more flexible in that case? 

Otherwise yes, please add a code comment! that would be super helpful in understanding this.